src/Utility/SecretGenerator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Accompli\Utility;
+
+/**
+ * Generatator to generate secrets.
+ *
+ * @author Ron Rademaker
+ */
+class SecretGenerator implements ValueGeneratorInterface
```

Comment:
I disagree, naming it according to its usage allows updating to a better secret-generation method in the future without having to update any other code (or ending up with a nonsense classname).